### ISSUE TITLE
Feature request: Support systemd user services e.g. via a CLI flag

### ISSUE THREAD

**Issue by ristomatti** [Lifailon/lazyjournal#8]
Currently it seems only system services/units can be listed. When `systemctl` or `journalctl` are used with `--user` flag, it lists only user specific services/units.

**Comment by Lifailon** [comment]
@ristomatti Have you tried using the right and left arrows in the `Unit list` window?

The second journal `System journals` contains a list of all journals with the filter `UNIT`, and the third journal `User journals` contains a list of all journals with the filter `USER_UNIT`.

Equivalent to the command:
```shell
journalctl -F UNIT
# and
journalctl -F USER_UNIT
```

As far as I know (did the analysis), these lists cover all possible unique journals in the system.

**Comment by ristomatti** [comment]
Oops sorry I just did a quick test if it runs at all :flushed:. The startup state with all the `dev-disk-id` listing (shown in the earlier screenshot) confused me and I just tested the "System var logs" listing for output. But should the search field still find the user specific units? I tried searching for two I've created and know they're running and the search came out empty.

**Comment by ristomatti** [comment]
(earlier screenshot referring to my closing comment on issue #7)

**Comment by ristomatti** [comment]
![image](https://github.com/user-attachments/assets/a05857f3-dde0-47fe-8e5c-a5b40a0d8402)

![image](https://github.com/user-attachments/assets/465d9e01-4953-442d-afc6-263ac6b595c1)

**Comment by ristomatti** [comment]
```
$ journalctl -F USER_UNIT | grep keymapper
keymapper.service
keymapper-state.service
```

**Comment by Lifailon** [comment]
The `Unit list` list only lists system services (as in the screenshot). Can you check the output in the `User journals` tab (by double-clicking the `right` key in the selected window)?

It should show two services, as in your output here:

> $ journalctl -F USER_UNIT | grep keymapper\nkeymapper.service\nkeymapper-state.service

**Comment by ristomatti** [comment]
Yup they found them. I just expected the search to look everywhere. Closing again, sorry for the noise!
